tests/testproject/TestProject.IntegrationServiceA/Redis/RedisExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using StackExchange.Redis;
+
+public static class RedisExtensions
+{
+    public static void MapRedisApi(this WebApplication app)
+    {
+        app.MapPost("/redis/{key}", SetKeyAsync);
+
+        app.MapGet("/redis/{key}", GetKeyAsync);
+    }
+
+    private static async Task<IResult> SetKeyAsync(string key, HttpContext context, IConnectionMultiplexer cm)
+    {
+        using var sr = new StreamReader(context.Request.Body);
+        var body = await sr.ReadToEndAsync();
```

Comment:
Looks like it only supports JSON:

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/parameter-binding?view=aspnetcore-8.0#binding-failures

Either that, or you can just have a `Stream body` parameter, which is the same as saying `context.Request.Body`.

So I guess this is more complicated than what it is worth. We can just keep the existing code.

Reply:
> Either that, or you can just have a Stream body parameter, which is the same as saying context.Request.Body.

I'd recommend `Stream body` but since this is just test code this is fine